### PR TITLE
Add toggle bool value command

### PR DESCRIPTION
**Pull request by mx990**
Add toggle bool value command and also add key binding for triggering events.

### CODE REVIEW

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml [hunk 1]
```diff
@@ -548,6 +548,12 @@
              id="org.eclipse.ui.commands.toggleState">
       </state>
     </command>
+    <command
+          categoryId="org.eclipse.fordiac.ide.commands.category"
+          description="Negate the value of a BOOL variable"
+          id="org.eclipse.fordiac.ide.application.commands.toggleBoolValue"
+          name="Toggle Value">
```
Comment (azoitl) [inline plugins/org.eclipse.fordiac.ide.application/plugin.xml]: please move to new plugin.properties

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml [hunk 2]
```diff
@@ -548,6 +548,12 @@
              id="org.eclipse.ui.commands.toggleState">
       </state>
     </command>
+    <command
+          categoryId="org.eclipse.fordiac.ide.commands.category"
+          description="Negate the value of a BOOL variable"
```
Comment (azoitl) [inline plugins/org.eclipse.fordiac.ide.application/plugin.xml]: please move to new plugin.properties

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml [hunk 3]
```diff
@@ -971,6 +1007,14 @@
           </and>
         </visibleWhen>
       </command>
+      <command
```
Comment (azoitl) [inline plugins/org.eclipse.fordiac.ide.application/plugin.xml:1024]: wouldn't it be better to have it in the edit menu part?

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml [hunk 4]
```diff
@@ -548,6 +548,12 @@
              id="org.eclipse.ui.commands.toggleState">
       </state>
     </command>
+    <command
+          categoryId="org.eclipse.fordiac.ide.commands.category"
+          description="Negate the value of a BOOL variable"
+          id="org.eclipse.fordiac.ide.application.commands.toggleBoolValue"
```
Comment (azoitl) [inline plugins/org.eclipse.fordiac.ide.application/plugin.xml:554]: please also add it to the source menu

**Review by azoitl — CHANGES_REQUESTED** [review]

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml (hunk 2 above)
Comment (mx990) [reply plugins/org.eclipse.fordiac.ide.application/plugin.xml]: Done

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml (hunk 1 above)
Comment (mx990) [reply plugins/org.eclipse.fordiac.ide.application/plugin.xml]: Done

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml (hunk 3 above)
Comment (mx990) [reply plugins/org.eclipse.fordiac.ide.application/plugin.xml:1024]: I also need it in the context menu of the variables view.

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml (hunk 4 above)
Comment (mx990) [reply plugins/org.eclipse.fordiac.ide.application/plugin.xml:554]: Done

File: plugins/org.eclipse.fordiac.ide.application/plugin.xml (hunk 3 above)
Comment (azoitl) [reply plugins/org.eclipse.fordiac.ide.application/plugin.xml:1024]: ah ok.

**Review by azoitl — APPROVED** [review]

**Pull request merged**